README.md

```diff
@@ -47,3 +53,16 @@ Stop/Down docker containers
 docker-compose stop # pauses containers
 docker-compose down # brings down containers
 ```
+
+File S3 Object Requests Format
```

Comment:
This should probably be documented in the respective file itself as something like a docustring example, etc.